jdbc-v2/src/test/java/com/clickhouse/jdbc/StatementTest.java

```diff
@@ -573,4 +573,47 @@ public void testSwitchDatabase() throws Exception {
             }
         }
     }
+
+    @Test(groups = { "integration" })
+    public void testNewLineSQLParsing() throws Exception {
+        try (Connection conn = getJdbcConnection()) {
+            String sqlCreate = "CREATE TABLE balance ( `id` UUID, `currency` String, `amount` Decimal(64, 18), `create_time` DateTime64(6), `_version` UInt64, `_sign` UInt8 ) ENGINE = ReplacingMergeTree PRIMARY KEY id ORDER BY id;";
+            try (Statement stmt = conn.createStatement()) {
+                int r = stmt.executeUpdate(sqlCreate);
+                assertEquals(r, 0);
+            }
+            try (Statement stmt = conn.createStatement()) {
+                String sqlInsert = "INSERT INTO balance VALUES (generateUUIDv4(), 'EUR', '42.42', now(), 144, 255);";
+                int r = stmt.executeUpdate(sqlInsert);
+                assertEquals(r, 1);
+            }
+            try (Statement stmt = conn.createStatement()) {
+                String sqlSelect = new StringBuilder("-- SELECT amount FROM balance FINAL;\n")
```

Comment:
can we add tests with a comment
- in the middle of a query
- at the end of the query